recipes/libdicom/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+meson setup --buildtype="release" --prefix="${PREFIX}" builddir
```

Comment:
```suggestion
meson setup ${MESON_ARGS} builddir
```

meson args is defined globally in conda-forge as
```
MESON_ARGS=--buildtype release --prefix=$PREFIX -Dlibdir=lib
```